roles/haproxy/templates/etc/haproxy/haproxy_openstack.cfg

```diff
@@ -69,7 +69,7 @@ frontend {{ name }}
 backend {{ name }}
 
   {% if name == "novnc" -%}
-  option httpchk GET /vnc_auto.html
+
```

Comment:
Since this is removing the only change for "novnc", we should remove it from the if statement and instead move ironic up to the top level if.

Without httpchk, what's the fallback for checking for health?